compact/far_forward/ZDC_SiPMonTile.xml

```diff
@@ -0,0 +1,120 @@
+<!-- SPDX-License-Identifier: LGPL-3.0-or-later -->
```

Comment:
We need a detector configuration that would include this file. It can be as simple as only your detector. It could be as simple as https://github.com/eic/epic/blob/main/configurations/dirc_only.yml, but would at least enable all the CI checks that we use to validate the geometry implementations.

Reply:
hi @veprbl, 
Thanks for this feedback.  I have added this in commit https://github.com/eic/epic/commit/44d304ce74b60251717ae26ef0799f76943c915e  as https://github.com/eic/epic/blob/ZDC_SiPMonTile/configurations/zdc_sipm_on_tile_only.yml  
Best regards,
Sebouh